package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "liquibase",
-  "version": "2.0.5",
+  "name": "@rimac-automobili/liquibase",
+  "version": "1.0.1",
```

Comment:
Must: We'll need to make sure that the version number is correct here.

Reply:
@ijakab lets fix these PR as suggested by @tabuckner (name, version etc...)

Reply:
Yes, sorry about this, we needed the change so I forked this in the meantime and forgot about this. Ok, reverting as it was.